### DISCUSSION TITLE
Status History: Allow ordering of labels

### DISCUSSION THREAD

**Discussion by Alan01252** [grafana/grafana#45440] [Dashboards, Panels & Visualizations]
The status history plugin works great, however, it's not possible to order the labels.

This is useful if using Prometheus to populate the status history panel, especially as the promql developers have taken a hard stance that label sorting should be solved at the presentation level instead of within Prometheus itself.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I think this will work if you use the join by field transformation, followed by the  organize fields by name transformation . Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
